tests/test_utils.py

```diff
@@ -313,3 +313,19 @@ def test_check_status_code_for_missing_status_code(capsys, repo_url):
 
     captured = capsys.readouterr()
     assert captured.out == "NOTE: Try --verbose to see response content.\n"
+
+
+@pytest.mark.parametrize(
+    ("filename, expected"),
+    [
+        (helpers.WHEEL_FIXTURE, "15.4 KB"),
+        (helpers.SDIST_FIXTURE, "20.8 KB"),
+        (helpers.NEW_SDIST_FIXTURE, "26.1 KB"),
+        (helpers.NEW_WHEEL_FIXTURE, "21.9 KB"),
+    ],
+)
+def test_get_file_size(filename, expected):
+    """Compare the size of a file to the expected value."""
```

Comment:
Test docstrings are an opportunity to describe the expected behavior. So, for example:

```suggestion
    """Get the size of file as a string with units."""
```

In this case, it's somewhat redundant with the function docstring, and personally, I would probably skip it for this test, but the project is pushing for more docstrings, so I think it's good to get in the habit.